packages/client/src/bags/types/putBagPromocodes.types.ts

```diff
@@ -0,0 +1,25 @@
+import type { Bag } from './bag.types';
+import type { Config } from '../..';
+
+export type PromoCodeInformation = {
+  promoCode: string;
+  isValid: boolean;
+  error: {
+    code: string;
+    message: string;
+  };
+};
+
+export type BagPromoCodesInformation = {
```

Comment:
Try to keep the naming consistent, there are some types that refer to `promoCode` (`PromoCodeInformation`, `BagPromoCodesInformation`) while other types are with `promocode` (`PutBagPromocodesData`, `PutBagPromocodes`). In my view, promocode should not be in camel case so this type should be named `BagPromocodesInformation`. Please, review any usages of promocode in the code to have this naming style.

Reply:
I will change this but the putBagPromocodes response from the API will give us an object with `promoCodesInformation`

Reply:
Yes, sorry, only change the naming where it is safe to do so. If the endpoint returns `promoCodesInformation`, we cannot change that specific name but everywhere else where it does not refer to names shared with the server, we can.

Reply:
Changed 👍

Reply:
In `packages/redux/src/bags/actions/factories/setBagPromocodesFactory.ts`, the entities name can be renamed to `bagPromocodesInformation` as well to keep consistency. The remaining ones, are correctly renamed, thanks!

Reply:
Fixed now @boliveira